packages/next-swc/crates/next-api/src/middleware.rs

```diff
@@ -198,7 +198,11 @@ impl Endpoint for MiddlewareEndpoint {
                 .await?
                 .clone_value();
 
-            Ok(WrittenEndpoint::Edge { server_paths }.cell())
+            Ok(WrittenEndpoint::Edge {
+                server_paths,
+                client_paths: vec![],
```

Comment:
middleware might have client path. e.g. if you use `new URL` to include an image. The image would be on client side